### PR TITLE
Docs/gcs plugin

### PR DESCRIPTION
**Pull request by garyluu**
For https://ucsc-cgl.atlassian.net/browse/SEAB-113

- Add short description + link to the gcs plugin.
- A chart summarizing plugins because there's quite a few now
- Removed 1.5.0 Dockstore stuff

Edit: I like https://stackedit.io/app# more.  Too bad it can't control the commit message and also puts a comment near the end.

### CODE REVIEW

File: _docs/advanced-features.md [hunk 1]
```diff
@@ -46,12 +46,25 @@ Provisioning for output files works in the same way and has been tested with S3
...
+
 ### AWS S3
 
 For AWS S3, create a `~/.aws/credentials` file and a `~/.aws/config` file as documented at the following [location](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html#cli-config-files).
 
 Get more information on the implementing plugin at [s3-plugin](https://github.com/dockstore/s3-plugin).
 
+### Google Cloud Storage
+
+For Google Cloud Storage, you can download, upload, and set metadata on uploaded objects with the gs-plugin.  The plugin handles urls with the `gs://` prefix such as `gs://genomics-public-data/references/GRCh38/chr1.fa.gz`. Get more information on the implementing plugin at https://github.com/dockstore/gs-plugin.
```
Comment (coverbeck) [inline _docs/advanced-features.md]: Make the gs-plugin repo a markdown link

File: _docs/advanced-features.md [hunk 2]
```diff
@@ -46,12 +46,25 @@ Provisioning for output files works in the same way and has been tested with S3
...
+| [synapse-plugin](https://github.com/dockstore/synapse-plugin)                         | syn://  | syn://syn8299856                                                                                 | download                                 |
+| [data-object-service-plugin](https://github.com/dockstore/data-object-service-plugin) | dos://  | dos://ec2-52-26-45-130.us-west-2.compute.amazonaws.com:8080/911bda59-b6f9-4330-9543-c2bf96df1eca | download                                 |
+
 ### AWS S3
 
 For AWS S3, create a `~/.aws/credentials` file and a `~/.aws/config` file as documented at the following [location](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html#cli-config-files).
 
 Get more information on the implementing plugin at [s3-plugin](https://github.com/dockstore/s3-plugin).
 
+### Google Cloud Storage
```
Comment (denis-yuen) [inline _docs/advanced-features.md]: Verify whether this is for 1.6.0 or if it existed already in 1.5.0

**Comment by garyluu** [comment]
Going to make a few more changes.

File: _docs/advanced-features.md (hunk 2 above)
Comment (garyluu) [reply _docs/advanced-features.md]: it exists for 1.5.0

Comment (coverbeck) [reply _docs/advanced-features.md]: It was only added in plugins.json for 1.6

Comment (garyluu) [reply _docs/advanced-features.md]: hmm...walt told me 1.5.0

**Comment by garyluu** [comment]
Leaving the stackedit comment in...too annoying to remove everytime.

**Pull request merged**